Project.toml

```diff
@@ -1,13 +1,13 @@
 name = "Oceanostics"
 uuid = "d0ccf422-c8fb-49b5-a76d-74acdde946ac"
 authors = ["tomchor <tomaschor@gmail.com>"]
-version = "0.6.2"
+version = "0.6.8"
```

Comment:
Why 0.6.8?

Reply:
Ah, typo. My bad. I must've confused with Oceananigans' version. I'll change it to 0.6.3 (since there are no new features).